### PR TITLE
Partnership TamperMonkey

### PR DESCRIPTION
**Pull request by PSouthern**
### Description of Work carried through

Adding the following TamperMonkey changes to support the Partnership organisation type:

- Partnership option added to Organisation dropdown
- Automate input on `partnership` page - default to `General Partnership`
- Automate input on `partnership-partners-list` page - Logic added to determine number of partners and select appropriate input
- Automate input on `partner-job-title` page
- Increment version number

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: tampermonkey/PPT_AutoComplete.js [hunk 1]
```diff
@@ -446,6 +457,14 @@ const partnerContactName = () => {
     }
 }
 
+const partnerJobTitle = () => {
+    if (currentPageIs('/register-for-plastic-packaging-tax/partner-job-title')) {
+
```
Comment (tmc-ee) [inline tampermonkey/PPT_AutoComplete.js:462]: Super picky but empty line 462 can be removed.

**Review by RyanKellyHMRC — APPROVED** [review]
LGTM

File: tampermonkey/PPT_AutoComplete.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         PPT Registration AutoComplete
 // @namespace    http://tampermonkey.net/
-// @version      15.8
+// @version      16.0
```
Comment (tmc-ee) [inline tampermonkey/PPT_AutoComplete.js:4]: Not sure how versioning is determined for the Tampermonkey scripts but this might be 15.9?

**Review by tmc-ee — APPROVED** [review]
A couple of extremely small comments but not show stoppers, looks great :+1:

**Review by paulblakehmrc — APPROVED** [review]

**Pull request merged**